renderMath.js

```diff
@@ -78,13 +87,12 @@ function renderMath(parentElement = "") {
             <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
 
             */
-            var polyfillScript = document.createElement("script");
-            polyfillScript.setAttribute("src", "https://polyfill.io/v3/polyfill.min.js?features=es6");
+            // var polyfillScript = document.createElement("script");
+            // polyfillScript.setAttribute("src", "https://polyfill.io/v3/polyfill.min.js?features=es6");
             var mathjaxScript = document.createElement("script");
-            mathjaxScript.setAttribute("id", "MathJax-script");
+            mathjaxScript.setAttribute("type", "text/javascript");
             mathjaxScript.setAttribute("async", "");
-            mathjaxScript.setAttribute("src", "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js");
-            document.head.appendChild(polyfillScript);
+            mathjaxScript.setAttribute("src", "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML");
```

Comment:
Is it possible to remove the attribute from the link? Adding a `?` and parameters to the link prevents caching, and caching speeds up the loading process. Our page is already really slow so having to download the mathjax script on every page load wouldn't be a help to that cause.
```suggestion
            mathjaxScript.setAttribute("src", "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js");
```